testsuites/CBLTester/topology_specific_tests/multiple_sync_gateways/test_sgreplicate_cbl.py

```diff
@@ -2118,7 +2118,9 @@ def test_sg_replicate_custom_conflict_resolve(params_from_base_test_setup, setup
     # Add merge js function to sgw config
     repl_id = "replication1"
     if external_js:
-        custom_conflict_js_function = "http://{}:5007/conflictResolver".format(get_local_ip())
+        # hosted js code statically as data center vms cannot reach mac machines , if these is not reachable, you need to restart js code
+        jscode_external_ip = "172.23.104.165"
```

Comment:
Do you want to get this IP from Environment variables, as should not expose our ips to outside. (Our git repo is exposed to outside)

Reply:
same comments here. I wonder what would be the best solution to hold sensitive values such as account used in testing or so.

Reply:
ips of data center is behind the firewall, ips cannot be accessed outside, wondering why it cannot be exposed and only for this.  We are already exposed few of the data center ips in some of the repo like server greenboard on server and mobile repos as anyways it cannot be accessed by outside